app/Dockerfile

```diff
@@ -1,21 +1,24 @@
-FROM ubuntu:16.04
+FROM alpine:3.6
 
 # Some ENV variables
 ENV PATH="/mattermost/bin:${PATH}"
 ENV MM_VERSION=4.4.1
 
 # Build argument to set Mattermost edition
-ARG edition=entreprise
+ARG edition=enterprise
 
-# Install some needed packages
-RUN apt-get update \
-    && apt-get -y --no-install-recommends install \
-      curl \
-      jq \
-      netcat \
-      ca-certificates \
-    && rm -rf /var/lib/apt/lists/* /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin
 
+RUN apk add --no-cache \
+	bash \
```

Comment:
If you use `/bin/sh` for the `app/entrypoint.sh` script then there is no need to install `bash` package I guess.